admin/metabox/class-metabox.php

```diff
@@ -901,11 +901,10 @@ public function enqueue() {
 			'log_level'               => WPSEO_Utils::get_analysis_worker_log_level(),
 		];
 
-		$woocommerce_conditional           = new WooCommerce_Conditional();
-		$woocommerce_active                = $woocommerce_conditional->is_met();
-		$wpseo_plugin_availability_checker = new WPSEO_Plugin_Availability();
-		$woocommerce_seo_file              = 'wpseo-woocommerce/wpseo-woocommerce.php';
-		$woocommerce_seo_active            = $wpseo_plugin_availability_checker->is_active( $woocommerce_seo_file );
+		$woocommerce_conditional = new WooCommerce_Conditional();
+		$woocommerce_active      = $woocommerce_conditional->is_met();
+		$woocommerce_seo_file    = 'wpseo-woocommerce/wpseo-woocommerce.php';
```

Comment:
I like how in the settings integration you used the Addon Manager to get the plugin file. Why do both approaches? 😉

Reply:
Oops right... Changed!